### PR TITLE
chore: cleanup projet

### PR DESCRIPTION
**Pull request by carlh98**
GitHub Issue: #

## Proposed Changes

chore: cleanup projet

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: src/library/ExtendedSplashScreen.Uno/ExtendedSplashScreen.Uno.csproj [hunk 1]
```diff
@@ -13,6 +13,13 @@
 		<Description>Nventive.ExtendedSplashScreen</Description>
 		<PackageId>Nventive.ExtendedSplashScreen.Uno</PackageId>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>extendedsplashscreen;splashscreen;async;mvvm;xamarin;uwp;ios;android</PackageTags>
+		<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+		<PackageProjectUrl>https://github.com/nventive/ExtendedSplashScreen</PackageProjectUrl>
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
+		<EmbedUntrackedSources>true</EmbedUntrackedSources>
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
```
Comment (jeanplevesque) [inline src/library/ExtendedSplashScreen.Uno/ExtendedSplashScreen.Uno.csproj:24]: This is for SourceLink. Is Source Link properly configured on this repo?

File: src/library/ExtendedSplashScreen.Uno/ExtendedSplashScreen.Uno.csproj [hunk 2]
```diff
@@ -24,6 +31,8 @@
 		<PackageReference Include="Uno.Core" Version="4.0.1" />
 		<PackageReference Include="Uno.UI" Version="4.0.13" />
 		<PackageReference Include="Uno.SourceGenerationTasks" Version="4.1.0" />
+		<!--Microsoft.SourceLink.GitHub is needed for Source Link support -->
```
Comment (jeanplevesque) [inline src/library/ExtendedSplashScreen.Uno/ExtendedSplashScreen.Uno.csproj:39]: I think you need more than that to configure Source Link.
Here's a full example: https://github.com/nventive/Chinook.DataLoader/pull/36/files
Adding Source Link also seems out of scope. It could be its own PR.

Comment (carlh98) [reply src/library/ExtendedSplashScreen.Uno/ExtendedSplashScreen.Uno.csproj:39]: The PR contains what is need for source link, will ask @vincentcastagna about the workaround

Comment (vincentcastagna) [reply src/library/ExtendedSplashScreen.Uno/ExtendedSplashScreen.Uno.csproj:39]: I'm not sure we need the workaround anymore. Can you try pack locally @carlh98 and check that the nupkg is fine in your nuget explorer ?

Comment (carlh98) [reply src/library/ExtendedSplashScreen.Uno/ExtendedSplashScreen.Uno.csproj:39]: I've verified the packages locally in the nuget explorer

**Review by jeanplevesque — APPROVED** [review]
